app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -0,0 +1,218 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import _root_.implicits.ImplicitDateFormatterImpl
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.viewmodels.CeaseIncomeSourcesViewModel
+
+@this(
+        mainTemplate: unifiedLayout,
+        implicitDateFormatter: ImplicitDateFormatterImpl,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link,
+        hr: hr,
+        govukTable: GovukTable
+)
+
+@(sources: CeaseIncomeSourcesViewModel,
+        isAgent: Boolean,
+        backUrl: String
+)(implicit messages: Messages,
+    user: auth.MtdItUser[_])
+
+@import implicitDateFormatter.longDate
+
+@getMessage(key: String, args: String*) = @{
+    messages(s"cease-income-sources.$key", args: _*)
+}
+
+@soleTraderBusinesses = @{ sources.soleTraderBusinesses }
```

Comment:
these 4 lines not really required and duplicating what we already have in the source view model;